docu/Concepts/TechnicalRequirements/BusinessEventProtocol.md

```diff
@@ -68,7 +68,7 @@ The following table lists for each event type the mapping between old and new ke
 | VISIT_GRADIDO                     | VisitGradidoEvent                      | x |  x  |     x     |        |        |              |              |          |        |
 | REGISTER                          | RegisterEvent                          | x |  x  |     x     |   x   |        |              |              |          |        |
 | LOGIN                             | LoginEvent                             | x |  x  |     x     |   x   |        |              |              |          |        |
-|                                   | VerifyRedeemEvent                      |    |      |          |        |        |              |              |          |        |
+| VERIFY_REDEEM                     | VerifyRedeemEvent                      |  x  |   x   |     x     |    x    |        |              |       (x)       |      (x)    |        |
```

Comment:
Ok that's a good idea to add here for the new eventtypes the new eventkeys.
But then please change the column headlines too
1. column: EventKey
2. column: EventType